### PR TITLE
Add args.task to determine the language

### PR DESCRIPTION
**Pull request by FrostML**
### PR types

Others

### PR changes

APIs

### Describe

Alter the key to "en" "de" and use `args.task` to choose the language.

### CODE REVIEW

File: benchmark/transformer/configs/transformer.base.yaml [hunk 1]
```diff
@@ -23,6 +23,8 @@ root: None
 use_gpu: True
 
 # Args for reader, see reader.py for details
+# The translation task to process.
+task: "en-de"
```
Comment (smallv0221) [inline benchmark/transformer/configs/transformer.base.yaml]: 跟glue统一一下用task_name吧

File: benchmark/transformer/reader.py [hunk 2]
```diff
@@ -57,9 +57,11 @@ def create_data_loader(args, places=None, use_all_vocab=False):
     args.src_vocab_size = padding_vocab(len(src_vocab))
     args.trg_vocab_size = padding_vocab(len(trg_vocab))
 
+    lang = args.task.split("-")
+
     def convert_samples(sample):
-        source = sample['source'].split()
-        target = sample['target'].split()
+        source = sample[lang[0]].split()
```
Comment (smallv0221) [inline benchmark/transformer/reader.py]: 这里这么处理的话，如果想反过来训练的话是吧lang的0和1反过来是么

Comment (FrostML) [reply benchmark/transformer/reader.py]: 可用 task 来控制. Done. Thanks.

File: benchmark/transformer/configs/transformer.base.yaml (hunk 1 above)
Comment (FrostML) [reply benchmark/transformer/configs/transformer.base.yaml]: Done. Thanks.

**Review by smallv0221 — APPROVED** [review]
LGTM

**Review by guoshengCS — APPROVED** [review]

**Pull request merged**